runtime/near-vm-runner/src/wasmtime_runner.rs

```diff
@@ -131,6 +131,17 @@ pub mod wasmtime_runner {
         let mut linker = Linker::new(&store);
         let mut logic =
             VMLogic::new(ext, context, wasm_config, fees_config, promise_results, &mut memory);
+        match logic.add_contract_compile_fee(code.len() as u64) {
```

Comment:
Same here. Also why couldn't we make this part of `VMLogic::new` to avoid code duplication?

Reply:
I was thinking about such approach, but it would mean that I cannot return `logic` as an outcome, which shall be done.